### PR TITLE
Refactor/styling buttons and ratings into modules

### PR DESCRIPTION
**Pull request by arielyxw**
Partially solves #62 

Refactored styling of food option buttons and ratings into separate modules.

### CODE REVIEW

**Review by bmar268 — APPROVED** [review]
LGTM, awesome work on this, is much clearer now!

File: munch-match/src/components/FoodOptionButton.jsx [hunk 1]
```diff
@@ -1,8 +1,10 @@
+import styles from "./FoodOptionButton.module.css"
+
 const FoodOptionButton = ({ foodOption, disabled, onClick }) => {
 
     return (
-        <button className={`button ${disabled ? "disabledButton" : ""}`} disabled={disabled} onClick={() => onClick(foodOption)}>
-            <img className={`food ${disabled ? "disabled" : ""}`} src={require(`../images/${foodOption}.png`)} alt={foodOption} />
+        <button className={`${styles.button} ${disabled ? "disabledButton" : ""}`} disabled={disabled} onClick={() => onClick(foodOption)}>
+            <img className={`${styles.food} ${disabled ? "disabled" : ""}`} src={require(`../images/${foodOption}.png`)} alt={foodOption} />
```
Comment (boxy8) [inline munch-match/src/components/FoodOptionButton.jsx]: `"disabled"` can be replaced with `styles.disabled` if you move those classes into the module.

Comment (boxy8) [reply munch-match/src/components/FoodOptionButton.jsx]: same with `disabledButton`

File: munch-match/src/App.css [hunk 2]
```diff
@@ -62,31 +62,7 @@ button {
...
-
-.food {
-  width: 160px;
-  height: 160px;
-  object-fit: cover;
-  overflow: hidden;
-}
-
-.surpise {
+.surprise {
```
Comment (boxy8) [inline munch-match/src/App.css]: ik you didn't add this class but it's unused so can we remove it hehe

Comment (boxy8) [reply munch-match/src/App.css]: dam also all anything with `App-logo`

**Review by boxy8 — CHANGES_REQUESTED** [review]
Just a few comments, nice job on moving to CSS module 😸

File: munch-match/src/App.css (hunk 2 above)
Comment (arielyxw) [reply munch-match/src/App.css]: Yep will remove those

File: munch-match/src/components/FoodOptionButton.jsx (hunk 1 above)
Comment (arielyxw) [reply munch-match/src/components/FoodOptionButton.jsx]: Nice catch thanks :)

**Review by boxy8 — APPROVED** [review]

**Pull request merged**